orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/general/FormattedAs.feature

```diff
@@ -4,7 +4,7 @@ Feature: User can specify that a value is so formatted
     Given the generation strategy is full
     And there is a field foo
 
-    @ignore #does not work with bigdecimal values
+    #commented out tests not working with big decimal values
```

Comment:
i'd rather keep them all ignored instead of having some coincidentally passing

Reply:
Tests removed and added as a comment on 1432